### PR TITLE
just added myself

### PR DESCRIPTION
**Pull request by tmamedbekov**
glad to be part of this

### CODE REVIEW

**Comment by emgoto** [comment]
The url needs to link to your `uses` page :)

**Comment by tmamedbekov** [comment]
okay let me fix that :)

**Comment by HugoDF** [comment]
Closing this until there's a "uses" page (doing it in a Gist is ok)

**Comment by tmamedbekov** [comment]
uses url has been added

File: src/data.js [hunk 1]
```diff
@@ -2064,6 +2064,26 @@ const pages = [
       'YouTuber',
     ],
   },
+  {
+    name: 'Tony Mamedbekov',
+    description:
+      'Architect, developer, cms enthusiast, technologoy evangelist, content creator, bloger, consultant',
+    url: 'https://www.tmamedbekov.dev',
```
Comment (palashmon) [inline src/data.js]: Please update the url to `https://tmamedbekov.dev/uses/` and commit again.

Comment (palashmon) [inline src/data.js]: Please update the url to `https://tmamedbekov.dev/uses/` and commit again. Thanks!

**Pull request closed without merging**